api/v1/mongodbcommunity_types.go

```diff
@@ -402,12 +423,39 @@ func (m MongoDBCommunity) GetScramOptions() scram.Options {
 		ignoreUnknownUsers = *m.Spec.Security.Authentication.IgnoreUnknownUsers
 	}
 
+	//Manage the Auth
```

Comment:
this comment is a little vague, I don't think it provides much value, we should include comments if it is not immediately clear what the code is doing.